hedgehog/test/test.hs

```diff
@@ -1,18 +1,10 @@
-import           Control.Monad (unless)
-import           System.IO (BufferMode(..), hSetBuffering, stdout, stderr)
-import           System.Exit (exitFailure)
+import           Hedgehog (runTests)
 
 import qualified Test.Hedgehog.Text
 
 
 main :: IO ()
-main = do
-  hSetBuffering stdout LineBuffering
-  hSetBuffering stderr LineBuffering
-
-  results <- sequence [
-      Test.Hedgehog.Text.tests
+main =
+  runTests
+    [ Test.Hedgehog.Text.tests
```

Comment:
Probably good to keep the same syntax style here, start of list on previous line that is

Reply:
:+1: Fixed the formatting in the comment for `runTests` as well.